lib/elixir/lib/application.ex

```diff
@@ -148,6 +145,23 @@ defmodule Application do
   end
 
   @doc """
+  Returns the value for `key` in `app`'s environment.
+
+  If the specified application is not loaded, or the configuration parameter
+  does not exist, raises `ArgumentError`.
+  """
+  @spec fetch_env!(app, key) :: value | no_return
+  def fetch_env!(app, key) do
+    case fetch_env(app, key) do
+      {:ok, value} -> value
+      :error ->
+        raise ArgumentError,
+          "application #{inspect app} is not loaded, " <>
```

Comment:
It took it verbatim from the `fetch_env/2` doc. :bowtie:
It depend on what "loaded" means, if it means "started" you are right; but I assume it means something like "that app is not in dependencies". Anyway it has to be clarified, do you have any suggestion?